vaccine_feed_ingest/runners/ny/am_i_eligible_covid19vaccine_gov/normalize.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import pathlib
+import re
+import sys
+
+CITY_RE = re.compile(r"^([\w ]+), NY$")
+# the providerName field smells like it's being parsed from someplace else,
+# a good number of them have leading \u1d42 and/or *, which we want to clean.
+# there's a bunch with a city name in them, but no real pattern to it, so
+# we'll leave that for now.
+NAME_CLEAN_RE = re.compile("^[\u1d42*]+")
+
+
+def normalize(site_blob: dict, timestamp: str) -> dict:
+    """
+    sample entry:
+
+    {"providerId": 1013, "providerName": "\u1d42**York College - Health and Physical Education Complex - Queens", "vaccineBrand": "Pfizer", "address": "Jamaica, NY", "availableAppointments": "Y", "isShowable": true, "lastUpdated": "2021-04-23T20:04:24"} # noqa: E501
+    """
+    name = NAME_CLEAN_RE.sub("", site_blob["providerName"]).strip()
+    city = CITY_RE.search(site_blob["address"]).group(1)
+    appts_available = True if site_blob["availableAppointments"] == "Y" else False
+    brand = site_blob["vaccineBrand"]
+
+    normalized = {
+        "id": f"am_i_eligible_covid19vaccine_gov:{site_blob['providerId']}",
+        "name": name,
+        "address": {
+            "city": city,
+            "state": "NY",
+        },
+        "availability": {
+            "appointments": appts_available,
+        },
+        "inventory": [
+            {
+                "vaccine": brand,
```

Comment:
CC: @rajbot / @bryanculbertson - do we have a defined enum for what these should normalize to? If not... let's add one!

No need to make any changes in this PR, we can address as a followup once the data schema is solidified.

Reply:
Yup, not a problem.